### ISSUE TITLE
🔦 Feature: Improve accessibility of the forms

### ISSUE THREAD

**Issue by b-g-h** [nocodb/nocodb#12691]
### Please confirm that the feature request does ***not*** already exist

- [x] I confirm there is no existing issue for this feature request.

### Use case

Forms with links to records cannot currently be used by blind users with active screenreader. The + button cannot be selected directly and has no label. 

### Suggested solution

This could be fixed by adding the role="button" and aria-label=“Link Record” in the div directly around the SVG of the + Symbol.  That would be a great improvement.
